generate-matrix

```diff
@@ -14,3 +14,7 @@ csv_file=matrix_$(date +%F_%T).csv
 ods_file=matrix_$(date +%F_%T).ods
 
 scp content-store-1.integration:/var/apps/content-store/dump.csv ./$csv_file
+
+echo Removing dump and dump script from integration
+
+rm ./dump-data && rm ./$csv_file
```

Comment:
Also this will remove the files from your local machine, not integration. You'll want to do something like `ssh content-store-1.integration 'rm ...'` like above.